src/main/java/kafdrop/service/KafkaMonitorImpl.java

```diff
@@ -250,10 +250,11 @@ public List<ConsumerVO> getConsumersByTopics(Collection<TopicVO> topicVos) {
   @Override
   public SearchResultsVO searchMessages(String topic,
                                         String searchString,
+                                        Integer partition,
                                         Integer maxmuimCount,
                                         Date startTimestamp,
                                         Deserializers deserializers) {
-    final var records = highLevelConsumer.searchRecords(topic, searchString, maxmuimCount, startTimestamp,
+    final var records = highLevelConsumer.searchRecords(topic, searchString, partition, maxmuimCount, startTimestamp,
```

Comment:
Since you are making a change here anyway, do you mind fixing the spelling error in `maximumCount` (now spelled as `maxmuimCount`)?